sale_order_line_menu/models/sale_order_line.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Moduon Team
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    date_order = fields.Datetime(
+        related="order_id.date_order",
+        readonly=True,
+        store=True,
+        index=True,
+    )
+    account_analytic_id = fields.Many2one(
```

Comment:
@EmilioPascual @rafaelbn I'm wondering if this shouldn't have named `order_analytic_account_id` ?

Reply:
@rousseldenis you're right, the name of this field wasn't the best. Perhaps in the future migration of the module the name should be changed.